packages/igniter-runtime/lib/-private/promise.ts

```diff
@@ -0,0 +1 @@
+export default window.Promise;
```

Comment:
@mixonic this is only true while backburner still exists, we will return RSVP to flushing as a microtask with igniter.